src/main/java/com/openshift/internal/restclient/model/template/Parameter.java

```diff
@@ -30,6 +30,12 @@ public Parameter(ModelNode node){
 		this.node = node;
 	}
 	
+	
+	@Override
+	public IParameter clone() {
```

Comment:
this needs to be declared on the interface. You can't call ITemplate.clone() as ITemplate is not an Object

current workaround is to call commons lang ObjectUtils.clone(object), which uses reflection to call clone()